hikari/impl/shard.py

```diff
@@ -818,16 +818,29 @@ async def _connect(self) -> typing.Tuple[asyncio.Task[None], ...]:
             url=url,
         )
 
-        # Expect initial HELLO
         hello_payload = await self._ws.receive_json()
-        if hello_payload[_OP] != _HELLO:
+        initial_op = hello_payload[_OP]
+
+        if initial_op == _RECONNECT:
+            # It is impossible that we receive RECONNECT as the initial opcode when the node we
```

Comment:
```suggestion
            # It is possible that we receive RECONNECT as the initial opcode when the node we
```

Unless I'm misinterpreting it?